### ISSUE TITLE
Is this package dead?

### ISSUE THREAD

**Issue by xitude** [laravel-notification-channels/webpush#196]
The last commit was 9 months ago, and with changes like iOS/Safari now allowing for [Push API](https://caniuse.com/push-api) and no support for PHP 8.1, I'm wondering if the package needs more support. Happy to provide it should the owner wish.

**Comment by joostdebruijn** [comment]
Yes, it's still alive. In the past months there were several releases to support new PHP and Laravel versions.
